src/transformers/models/mask2former/modeling_mask2former.py

```diff
@@ -0,0 +1,2416 @@
+# coding=utf-8
+# Copyright 2022 Meta Platforms, Inc.s and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch Mask2Former model."""
+
+import math
+import warnings
+from dataclasses import dataclass
+from numbers import Number
+from typing import Dict, List, Optional, Tuple
+
+import numpy as np
+import torch
+import torch.nn.functional as F
```

Comment:
Please use nn.functional directly in the code instead of one-letter imports

Reply:
Ah, just realized we can't import it as torch.nn.functional.F, that's why it's imported with `import torch.nn.functional as F` across other models

Reply:
Seems like this comment was set to resolved but I still see the one-letter import :)